docs/api.md

```diff
@@ -110,10 +110,11 @@ metadata:
 spec:
   replicas: 3
   version: v1.17.0
```

Comment:
Maybe bump this to a supported version?
```suggestion
  version: v1.20.0
```

Reply:
I am planning to update all docs and yaml files under `/examples ` with new version of API for both CRs' and CAPI objects in the separate commit or patch, thanks for reminding.

Reply:
@furkatgofurov7 if I remember correctly, @wgslr is doing that in some PR

Reply:
@kashifest nope, we have a PR for parameterization of e2e yaml files, but not updating the docs/examples with new API version.